integration_tests/tests/match/match.cy.ts

```diff
@@ -33,9 +38,31 @@ context('Placement Requests', () => {
     listPage.clickFindBed(placementRequests[0])
 
     // Then I should be taken to the search page
-    const searchPage = Page.verifyOnPage(SearchPage)
+    const searchPage = Page.verifyOnPage(SearchPage, person.name)
 
     // And I should see the search results
+    let numberOfSearches = 0
     searchPage.shouldDisplaySearchResults(bedSearchResults)
+    numberOfSearches += 1
+
+    const newSearchParameters = bedSearchParametersFactory.build()
+
+    // When I enter details on the search page
+    searchPage.changeSearchParameters(newSearchParameters)
+    searchPage.clickSubmit()
+    numberOfSearches += 1
+
+    cy.task('verifySearchSubmit').then(requests => {
+      expect(requests).to.have.length(numberOfSearches)
+
+      const body = JSON.parse(requests[1].body)
```

Comment:
Should we verify the payload of both requests? And should `body` have a more descriptive name, i.e. `newSearchRequestBody` or something?